catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/transformer/GmdTransformer.java

```diff
@@ -298,7 +307,8 @@ private void addMetacardCrs(final XstreamPathValueTracker pathValueTracker,
         crs.append(StringUtils.defaultString(pathValueTracker.getFirstValue(toPath(GmdMetacardType.CRS_VERSION_PATH))));
         crs.append(COLON);
         crs.append(StringUtils.defaultString(pathValueTracker.getFirstValue(toPath(GmdMetacardType.CRS_CODE_PATH))));
-        metacard.setAttribute(GmdMetacardType.GMD_CRS, crs.toString());
+        // TODO should this be location.crs-code?
```

Comment:
Without the version, this would map to crs-code.  `GmdMetacardType.CRS_AUTHORITY_PATH`:`GmdMetacardType.CRS_CODE_PATH` would be `EPSG:4326`.  I'm not sure if there is any value in parsing the version here at all.  Thoughts?